.github/workflows/database-backup.yml

```
@@ -129,6 +129,29 @@ jobs:
           node backend/scripts/verify-backup.js --backup "$BACKUP_FILE" > /tmp/verification-report.json
           cat /tmp/verification-report.json
 
+      - name: Verify SHA-256 integrity sidecar (WS4 / #1100)
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2012:info:12:11: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)